src/components/Flashcard/CardContent.js

```diff
@@ -1,45 +1,74 @@
 import React, { useState, useEffect } from 'react';
 import styles from './Flashcard.module.css';
-import { FaQuestion, FaCommentDots, FaSave, FaCheck, FaTimes } from 'react-icons/fa';
 
 import { Button } from 'components';
 
-export function CardContent({ question, addToUserBase, known, unknown}) {
-    const [isKnownAdded, setIsKnownAdded] = useState('iconButton');
-    const [isUnknownAdded, setIsUnknownAdded] = useState('iconButton');
+import { CrossIcon, CheckIcon, SaveIcon, AnswerIcon, QuestionIcon } from '../icons';
+
+export function CardContent({ question, addToUserBase, known, unknown }) {
+    const notActiveUserIconColor = '#416071';
+    const activeUserIconColor = '#9e005d';
+    const [isKnownAdded, setIsKnownAdded] = useState(notActiveUserIconColor);
+    const [isUnknownAdded, setIsUnknownAdded] = useState(notActiveUserIconColor);
 
     useEffect(() => {
         isClicked(known, setIsKnownAdded);
         isClicked(unknown, setIsUnknownAdded);
     });
 
-    const isClicked = (base, setClass) => {
-        if(base.includes(question.id)){
-            setClass('iconButton iconButtonActive')
-        }else{
-            setClass('iconButton')
+    const isClicked = (base, setColor) => {
+        if (base.includes(question.id)) {
+            setColor(activeUserIconColor);
+        } else {
+            setColor(notActiveUserIconColor);
         }
-    }
+    };
 
     return (
         <div className="flashcardPlusUserButtons">
             <div className={styles.flashcard}>
                 <div className={styles.flashcardInner}>
                     <div className={styles.flashcardFront}>
-                        <i><FaQuestion /></i>
+                        <QuestionIcon className={styles.flashcardIcon} />
                         <p>{question.question}</p>
                     </div>
                     <div className={styles.flashcardBack}>
-                        <i><FaCommentDots /></i>
+                        <i>
+                            <AnswerIcon className={styles.flashcardIcon} />
+                        </i>
                         <p>{question.answer}</p>
                     </div>
                 </div>
             </div>
             <div className={styles.flashcardUserButtons}>
-                <Button className={'iconButton'}><i><FaSave /></i></Button>
-                <Button className={isKnownAdded} onClick={() => addToUserBase(question.id, 'known')} ><i><FaCheck /></i></Button>
-                <Button className={isUnknownAdded} onClick={() => addToUserBase(question.id, 'unknown')} ><i><FaTimes /></i></Button>
+                <Button className={'iconButton'}>
+                    {' '}
```

Comment:
are these `{' '}` necessary?

Reply:
> are these `{' '}` necessary?

Nah, I commented something there and left that by mistake. I'll delete it before merge 👍